templates/1.0.0/templates/_helpers.tpl

```diff
@@ -5,9 +5,14 @@
 {{- define "fullname" -}}
   {{- $global := index . 0 -}}
   {{- $index := index . 1 -}}
+  {{- $metadata := $global.Values._config._metadata -}}
   {{- $release := $global.Release.Name | trunc 30 | trimSuffix "-" | lower -}}
   {{- $chart := $global.Chart.Name | trunc 20 | trimSuffix "-" | lower -}}
-  {{- printf "%s-%s-%d" $release $chart $index -}}
+  {{- if and $metadata (hasKey $metadata "revision") -}}
+    {{- printf "%s-%s-v%.0f-%d" $release $chart $metadata.revision $index -}}
```

Comment:
Are you converting metadata.revision to float when passing to the template?

Reply:
NO. It is done by template render. The render just use the `json` package to unmarshal json into a map. So numbers are `float64`.

Reply:
LGTM. Just make sure it is always float, otherwise we'll get string name.